ckan/cli/generate.py

```diff
@@ -77,3 +80,44 @@ def extension(output_dir):
 
     cookiecutter(template_loc, no_input=True, extra_context=context,
                  output_dir=output_dir)
+
+
+@generate.command(name=u'config', short_help=u'Create config from template.')
+@click.option(u'-o', u'--output-dir', help=u'Location to put the generated '
+                                           u'template.')
+def make_config(output_dir):
+    if output_dir is None:
+        print('\nERROR: Try again with \'-o path/to/ckan.ini\'')
+        sys.exit(1)
+
+    name = 'ckan.ini'
+
+    if name not in output_dir:
```

Comment:
We will default to `ckan.ini` on our installation instructions and deployment scripts but this name should not be enforced.
People might want to use the old `development.ini` / `production.ini` to support existing deployment pipelines, or use different ini files for different environments.

Reply:
That's a good point. No problem!